NitroxModel/DataStructures/GameLogic/PDASaveData.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using UnityEngine;
+using System.Collections.Generic;
+using ProtoBuf;
+
+namespace NitroxModel.DataStructures.GameLogic
+{
+    [Serializable]
+    [ProtoContract]
+    public class PDASaveData
+    {
+
+        [ProtoMember(1)]
+        public PDADataPartial PDADataPartial { get; set; }
+        [ProtoMember(2)]
+        public PDADataComplete PDADataComplete { get; set; }
+        [ProtoMember(3)]
+        public PDADataEnciclopedia PDADataEnciclopedia { get; set; }
+        [ProtoMember(4)]
+        public PDADataknownTech PDADataknownTech { get; set; }
+
+        public PDASaveData()
```

Comment:
We can probably remove this.  I don't think we need an explicit default constructor if there are no other parameterized constructors.  (if so, please do on other classes throughout the PR)